app/components/add-validations-to-form.js

```diff
@@ -0,0 +1,74 @@
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { GRAPHS } from '../controllers/formbuilder/edit';
+import { ForkingStore } from '@lblod/ember-submission-form-fields';
+import { getLocalFileContentAsText } from '../utils/get-local-file-content';
+import { FORM, RDF } from '../utils/rdflib';
+import { areValidationsInGraphValidated } from '../utils/validation-shape-validators';
+
+export default class AddValidationsToFormComponent extends Component {
+  @tracked builderStore;
+  @tracked builderForm;
+  @tracked formTtlCode;
+  @tracked showRdfForm = false;
+  @tracked sourceNode;
+
+  graphs = GRAPHS;
+  form_ttl_path = '/forms/validation/form.ttl';
+  meta_ttl_path = '/forms/validation/meta.ttl';
+  REGISTERED_VALIDATION_FORM_TTL_CODE_KEY = 'validationFormTtlCode';
+
+  constructor() {
+    super(...arguments);
+
+    if (!this.args.formTtlCode || this.args.formTtlCode == '') {
+      throw `Cannot add validations to an empty form`;
+    }
+
+    this.formTtlCode = this.args.formTtlCode;
+    this.sourceNode = this.args.sourceNode;
+
+    this.createBuilderStore().then((builderStore) => {
```

Comment:
I would move this to an async method or an ember-concurrency task and simply start it in the constructor.